Projects/Feature/InputInformationFeature/Sources/Intent/InputInformationIntent.swift

```diff
@@ -68,9 +71,31 @@ final class InputInformationIntent: InputInformationIntentProtocol {
                     profileImgURL: profileImageURL,
                     region: inputWorkInfo.workRegion,
                     salary: inputWorkInfo.salary,
-                    techStack: inputProfileInfo.techStack
+                    techStack: inputProfileInfo.techStack,
+                    projects: state.projects.concurrentMap {
+                        async let imageURL = self.imageUploadUseCase.execute(
+                            image: $0.iconImage?.data ?? .init(),
+                            fileName: $0.iconImage?.name ?? ""
+                        )
+                        let previewImages = try await $0.previewImages.concurrentMap {
+                            async let previewImageURL = self.imageUploadUseCase.execute(
+                                image: $0.data,
+                                fileName: $0.name
+                            )
+                            return try await previewImageURL
+                        }
```

Comment:
```suggestion
                        )
                        async let previewImages = $0.previewImages.concurrentMap {
                            async let previewImageURL = self.imageUploadUseCase.execute(
                                image: $0.data,
                                fileName: $0.name
                            )
                            return try await previewImageURL
                        }
```
으로 가는거 어떤가요, 마지막 return 할때 실행되도록하는거용

Reply:
오 저런 방법이 있었네요